tests/integration/root_ca_rotation_test.go

```diff
@@ -0,0 +1,233 @@
+package integration
```

Comment:
This is major feature, we should have both integration and e2e tests. Please take a look into https://github.com/etcd-io/etcd/issues/13637.on how to write one code that with run both.